src/libs/actions/Policy.ts

```diff
@@ -1306,7 +1306,7 @@ function createPolicyExpenseChats(policyID: string, invitedEmailsToAccountIDs: I
         const cleanAccountID = Number(accountID);
         const login = PhoneNumber.addSMSDomainIfPhoneNumber(email);
 
-        const oldChat = ReportUtils.getChatByParticipantsAndPolicy([sessionAccountID, cleanAccountID], policyID);
+        const oldChat = ReportUtils.getPolicyExpenseChat(cleanAccountID, policyID);
```

Comment:
Can we delete `ReportUtils.getChatByParticipantsAndPolicy` since it isn't in use anymore?